lib/id3c/api/routes.py

```diff
@@ -144,6 +144,30 @@ def receive_fhir(*, session):
 
     return "", 204
 
+@api_v1.route("/warehouse/verify-barcode-uses", methods=['POST'])
```

Comment:
It's not clear to me why this is nested under `/warehouse/…` when it's not a `warehouse` resource but an abstract RPC endpoint (what this [useful guide on naming RESTful resources](https://restfulapi.net/resource-naming/) calls the "controller" archetype).

Reply:
Ok, my only reason for putting it there is that it's interfacing with warehouse tables, but a new category makes sense. `/verify/…` might be a good option.

Reply:
I think typically if we want to use that controller pattern, we may want to name it with the verb at the end, something like `/identifier-use/verify` or `/barcode-use/verify`

Alternatively, we could restructure this enpoint a bit if we want it to be a more straightforward GET request for resources. Rather than doing the "verification" in the API, we could instead provide an endpoint that took the identifier, and returned the associated identifier use. That could be something like `/warehouse/identifiers/{id}/identifier-use` . I know you already discussed this with Jeremy though and I'm guessing that led to doing the verification within the API?

Reply:
My thought process with `/verify/…` (or I could rename it to '/verification/…') was to have a grouping that we could add other endpoints to down the road. The [current endpoints](https://backoffice.seattleflu.org/production/api) are grouped into `/receiving/…` and `/warehouse/…` so I was trying to stick with a similar pattern.

Jeremy's original request was for one endpoint to verify a single barcode use and another endpoint to verify a list of barcodes and uses.  After chatting with him he agreed that for single barcodes he could just use the GET `/v1/warehouse/identifier/<id>` since that now also returns the `use` value. So this PR is just to add an endpoint for verifying a list, and he said his preferred response format is a list in the same order as the input, with a status for each barcode and use.

Reply:
Ok, that's fair if Jeremy wants this in addition to  `/v1/warehouse/identifier/<id>`, we can definitely add this then.

In regards to naming, I'm inclined to say that existing endpoints were grouped with `warehouse` and `receiving` prefixes to reuse the namespacing from the database. For this type of API, to follow naming conventions for RESTful APIs, I believe we'd want it to end in a verb to differentiate from endpoints that relate to documents or collections (see https://restfulapi.net/resource-naming/). If you'd like to namespace under `verification`, I think that would leave us with  `/verification/identifier-uses/verify` or `verification/barcode-uses/verify`.

Reply:
That makes sense, I'll go ahead and rename it.  Jeremy is only going to be passing barcodes, but to keep things consistent with the `identifier` endpoints, I'll update it to accept full uuids or shortened barcodes and go with `/verification/identifier-uses/verify` for the name.

Reply:
Tested a few different way but performance of the SQL query was much worse when trying to accommodate full UUIDs and shortened barcodes (since the input list could potentially contain either or both), so I left it as-is was to only check shortened barcodes. 

I renamed the endpoint `verification/barcode-uses/verify` as suggested and updated API documentation page accordingly.